tsconfig.json

```diff
@@ -1,12 +1,13 @@
 {
   "compilerOptions": {
     "target": "es2019",
-    "module": "commonjs",
+    "module": "Node16",
```

Comment:
Why node 16?

Reply:
Node16 / NodeNext are the recommended modules for modern node apps. Happy to change to NodeNext if you'd prefer :)

https://www.typescriptlang.org/docs/handbook/modules/theory.html#the-module-output-format

Reply:
i didn't realise it was '16+' not just 16, looks all good to me